packages/pwa/app/pages/checkout/partials/shipping-address-selection.jsx

```diff
@@ -349,7 +353,9 @@ ShippingAddressSelection.propTypes = {
     selectedAddress: PropTypes.object,
 
     /** Override the submit button label */
-    submitButtonLabel: PropTypes.string,
+    submitButtonLabel: PropTypes.shape({
+        defaultMessage: PropTypes.oneOfType([PropTypes.string, PropTypes.array])
```

Comment:
In which case it would be a string? 🤔 , afaik, `defaultMessage` has always been an object. 
```suggestion
        defaultMessage: PropTypes.oneOfType([PropTypes.string, PropTypes.object])
```

Reply:
`defaultMessage` is inside an object. `defaultMessage` itself is written as a string. 
```js
{id: '...', defaultMessage: 'hello world'}
```
But our localization setup uses a _compiled_ version of the translations. This results in `defaultMessage` in array form. Maybe I'll add a comment in the prop types to make it clearer.